.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: go
+
+go:
+  - 1.10.x
+
+services:
+  - docker
+
+before_install:
+  - set -e
+  #TODO this is very insane, find a long term way to have dep available (in a safe way):
+  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Replace this with:
```
 curl -L https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 -o $GOPATH/bin/dep
 echo "31144e465e52ffbc0035248a10ddea61a09bf28b00784fd3fdd9882c8cbb2315  dep" > $GOPATH/bin/checksums
 cd $GOPATH/bin
 sha256sum -c checksums
```
This will pin it to a stables checked version. I don't know how to run Travis CI builds with preinstalled tools.

Reply:
According to @bechtoldt will be fixed in #3